c-c++.md

```diff
@@ -37,7 +32,7 @@ Starred version marks the default system compiler.
 Distribution    | GCC                  | Clang/LLVM
 ----------------|----------------------|-------------
 Amazon Linux 2  | 7*, 10               | 7, 11*
-Ubuntu 20.04    | 7, 8, 9*, 10         | 6, 7, 8, 9, 10, 11
+Ubuntu 20.04    | 7, 8, 9*, 10         | 11, 12, 13, 14*
```

Comment:
Ubuntu 20.04 only provides the following versions of Clang: 6, 7, 8, 9, 10, 11, 12.

Ubuntu 22.04 provides clang 11, 12, 13, 14 with default clang-14 as in the above change.
I would recommend adding a new line for Ubuntu 22.04.
Ubuntu 22.04 provides gcc versions 9, 10, 11, 12, with default gcc-11.

Reply:
thanks, fixed.